learn-ocaml-client.opam

```diff
@@ -36,6 +36,7 @@ depends: [
   "ppx_tools"
   "ppx_sexp_conv" {= "v0.9.0"}
   "ppx_fields_conv" {= "v0.9.0"}
+  "yojson" {>= "1.4.0" }
```

Comment:
Why add a dependency on yojson instead of using ezjsonm?  FWIW yojson is part of learn-ocaml’s dependencies but before this commit it looks like no binaries are even linked to it…

Reply:
@agrn good catch! Indeed it seems learn-ocaml currently depends on these two JSON libraries…

because even if this PR added yojson in learn-ocaml-client.opam, it was already part of learn-ocaml.opam:

https://github.com/ocaml-sf/learn-ocaml/blob/b3d0e756c62f1813bebab29916443c412458e372/learn-ocaml.opam#L51

but it indeed seems yojson was not actually used.

@yurug do you agree with the proposed choice? (removing yojson from learn-ocaml direct dependencies, and rewrite #354 using ezjsonm)

Reply:
I agree!